Core/clim-core/panes.lisp

```diff
@@ -363,11 +363,38 @@ order to produce a double-click")
   (print-unreadable-object (pane sink :type t :identity t)
     (prin1 (pane-name pane) sink)))
 
+(defgeneric find-concrete-pane-class (pane-realizer pane-type)
```

Comment:
Another observation: it will probably be a common case that a client cannot continue when no class is found. For that case (and probably as the default behavior), it would be good to signal an error when no suitable class can be found (similar to `cl:find-class`).

I suggest either `&optional errorp` or `&key if-does-not-exist` to control the behavior.

Reply:
makes sense, I'll add &optional errorp

Reply:
fixed